content/content-strategy/cms/cms-requirements/example.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: text/callout
+---
+The templates below are examples that you can tailor to suit your needs.
+  * [NSW Department of Education: content audit spreadsheet](/assets/files/content-strategy/nsw-education-content-audit-spreadsheet.xlsx) (XLSX 191KB)
+  * [Department of Agriculture: content audit spreadsheet](/assets/files/content-strategy/agriculture-example-audit-spreadsheet.xlsx) XLSX 146KB)
+  * [UK Government Digital Service: content audit spreadsheet](/assets/files/content-strategy/govuk-example-audit-sheet.xlsx) XLSX 44KB)
```

Comment:
Not sure if this is the new location for the removed spreadsheet by the same name. I don't recall seeing the file being readded at a new location. cc @Phend-117

Reply:
Steve. Removed. Not relevant.